packages/jasmine/test/BUILD.bazel

```diff
@@ -30,3 +30,13 @@ jasmine_node_test(
     jasmine = "@npm//jasmine",
     shard_count = 3,
 )
+
+jasmine_node_test(
+    name = "coverage_test",
+    srcs = [
+        "coverage.spec.js",
+        "coverage_source.js",
+    ],
+    coverage = True,
+    jasmine = "@npm//jasmine",
```

Comment:
will need to rebase and add `deps = ["//:jasmine_runner"],` to this target

Reply:
Can you also add a smoke test to `/e2e/jasmine`? It would test the `v8-coverage` can be required in user land when using the `@bazel/jasmine` package.
A test which verifies the report works would be good but I think we can merge this without one and add that in a follow up PR.

Reply:
done